.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,7 +6,7 @@
 - [ ] Change is maintainable (easy to change, telemetry, documentation).
 - [ ] [Library release note guidelines](https://ddtrace.readthedocs.io/en/stable/contributing.html#Release-Note-Guidelines) are followed.
 - [ ] Documentation is included (in-code, generated user docs, [public corp docs](https://github.com/DataDog/documentation/)).
-- [ ] PR description includes explicit acknowledgement/acceptance of the performance implications of this PR as reported in the benchmarks PR comment.
+- [x] OPTIONAL: PR description includes explicit acknowledgement of the performance implications of the change as reported in the benchmarks PR comment.
```

Comment:
```suggestion
- [ ] OPTIONAL: PR description includes explicit acknowledgement of the performance implications of the change as reported in the benchmarks PR comment.
```
I'd prefer reviewers/authors remember to manually check this off rather than have this checked off by default, since having this pre-checked off might lead to people forgetting. WDYT?

Reply:
My thinking with the way I wrote this is that if it's marked as "optional" but CI won't pass unless it's checked off, then it's not really optional.

Reply:
On the other hand, technically it isn't optional, it's only "optional" when the change itself doesn't have any performance implications to acknowledge (i.e. chore/docs/ci-related PR). Maybe we should add that clause in the checklist item, i.e. `PR description includes explicit acknowledgement of the performance implications of the change as reported in the benchmarks PR comment. (Optional if PR involves chore/docs/ci changes)` or something along those lines.

Reply:
I misinterpreted what the purpose of this change was, in that case I'm fine with this. Can you update the PR description to explain why we're marking this as optional to clarify for future readers? Thanks!